.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: false
+language: go
+go:
+  - "1.11.x"
+  - "1.10.x"
+  - "1.9.x"
+  - master
+matrix:
+  allow_failures:
+    - go: master
+  fast_finish: true
+script:
+  - go get -t -v ./...
+  - diff -u <(echo -n) <(gofmt -d -s .)
+  - go tool vet .
+  - go test -v -race ./...
```

Comment:
Shall we have this in a Makefile?

Reply:
Yeah makes sense but wanted a quick solution for now. 

Beside Makefile we also should vendor dependencies and Test dependencies. 

I’ve ignored that for now to just getting started with CI asap

Reply:
Sounds good to start with :+1: